core/scripts/mercury/mercury.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"sync/atomic"
+	"time"
+)
+
+var reportsCount uint64
+
+func handleReports(w http.ResponseWriter, req *http.Request) {
+	if req.Method != "POST" {
+		fmt.Printf("%s is not supported\n", req.Method)
+		return
+	}
+
+	atomic.AddUint64(&reportsCount, 1)
+
+	// fmt.Println("POST /reports called")
```

Comment:
we planning on removing these comments?

Reply:
Not sure, its just a script. Could be useful to leave them for debugging?